lib/v2/screens/settings/edit_name/edit_name_screen.dart

```diff
@@ -60,5 +90,12 @@ class _EditNameScreenState extends State<EditNameScreen> {
     _editNameBloc.add(OnNameChanged(name: _nameController.text));
   }
 
-  void _onSubmitted() {}
+  void _onSubmitted() {
+    if (_formKeyName.currentState.validate()) {
+      _editNameBloc.add(SubmitName(
+          accountName: SettingsNotifier.of(context).accountName,
+          privateKey: SettingsNotifier.of(context).privateKey,
+          nodeEndpoint: SettingsNotifier.of(context).nodeEndpoint));
```

Comment:
I dont like we are passing this data from the UI. 
But until get rid of Settings Notifier we dont really have an option.

Reply:
I know